### DISCUSSION TITLE
How to override the styles of a default control?

### DISCUSSION THREAD

**Discussion by LZD121** [AvaloniaUI/Avalonia#10730] [Q&A]
I have a style for "Button.avv". This default style works properly. 
![image](https://user-images.githubusercontent.com/59716053/226787772-0abe858e-06bd-4022-8bd6-f1940bedd42b.png)
![image](https://user-images.githubusercontent.com/59716053/226787821-c3953f7e-5f14-429d-ba5a-0983f0434649.png)

But when I add a style "Button.avv:pointerover", I found it doesn't work.
The style on pointerover still remain the original style of Button?
![image](https://user-images.githubusercontent.com/59716053/226787862-b8e49e47-832a-4b60-8bab-43c93018ac49.png)
![image](https://user-images.githubusercontent.com/59716053/226787901-2d6bd27d-d246-43c0-b33c-8f0692552a97.png)

How can I override styles of a default control?

**Comment by jeff9571** [comment]
how about changing the button pointerover style to `<Style Selector="Button.avv /template/ ContentPresenter">` or `<Style Selector="Button.avv:pointerover /template/ ContentPresenter#PART_ContentPresenter">` ?

**Comment by timunie** [comment]
@LZD121 please read https://docs.avaloniaui.net/docs/styling/troubleshooting 

Moreover it's always worth a look into the original styles. Assuming you are using Avalonia 0.10.18, here is the original button style: 

https://github.com/AvaloniaUI/Avalonia/blob/401bc521d2947bf7699a530f3d6e43e7a32c6baf/src/Avalonia.Themes.Fluent/Controls/Button.xaml#L44-L49
